universal-login-commons/test/core/utils/InitializationHandler.unit.test.ts

```diff
@@ -0,0 +1,110 @@
+import chai, {expect} from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import {InitializationHandler} from '../../../src/core/utils/InitializationHandler';
+
+chai.use(sinonChai);
+
+describe('UNIT: InitializationHandler', () => {
+  describe('sync', () => {
+    const initialize = sinon.stub();
```

Comment:
Make those into a promises that resolve after some time (like 10 ms) to test for race conditions